frontend/language/src/en.json

```diff
@@ -242,7 +242,7 @@
   "general.disabled": "Disabled",
   "general.edit": "Edit",
   "general.enabled": "Enabled",
-  "general.error_message": "An error has occurred!",
+  "general.error_message": "An error has occurred! If the problem persists, please <a>contact us</a>.",
```

Comment:
Is this right? There is no href here, and the text does not seem to be treated as html:
![Screenshot 2023-11-22 at 14 10 41](https://github.com/Altinn/altinn-studio/assets/1636323/c17ab820-6a39-4d63-93d2-238fb3c71101)

Reply:
I'm not sure what the problem is here 🤔 The link works on my machine, and I couldn't reproduce the issue.

At first, I thought it was a caching issue, so I deleted the `node_modules` folder, ran `yarn cache clean`, and then `yarn`, but the link was still working.

I then removed the entire altinn-studio folder, and all containers, images, and volumes from Docker. Then, I did a `git clone` and reinstalled everything using `yarn && yarn setup`. I tested the website through Docker and using webpack-dev-server and also tested on Chrome, Firefox, and Safari, but I couldn't reproduce the issue.

<img width="497" alt="link" src="https://github.com/Altinn/altinn-studio/assets/24462611/8b22c392-72cb-4c39-94b1-7afa5142e5bd">

Reply:
Could you try reinstalling all the packages by removing the `node_modules` folder and running `yarn`?

Reply:
Was not able to reproduce the issues after a full re-install and build 😊